dashboard/src/actions/apps.ts

```diff
@@ -8,12 +8,17 @@ import { hapi } from "../shared/hapi/release";
 import { definedNamespaces } from "../shared/Namespace";
 import { validate } from "../shared/schema";
 import {
+  CreateError,
+  DeleteError,
+  FetchError,
   IAppOverview,
   IChartUpdateInfo,
   IChartVersion,
   IRelease,
   IStoreState,
+  RollbackError,
   UnprocessableEntity,
+  UpgradeError,
```

Comment:
Just a thought as I see this repeated, we could instead have a `shared/errors` then here just do `import * as errors from "shared/errors";` , then within the code just use `errors.CreateError` for example. Fine either way.